scripts/spot-grid.py

```diff
@@ -224,7 +271,11 @@ def main(args):
     handler.setFormatter(logging.Formatter('%(asctime)s (%(module)s) %(levelname)s: %(message)s'))
     logging.getLogger().addHandler(handler)
 
-    if options.role:
+    if options.filename:
+        logging.info("Will process image file %s" % options.filename)
```

Comment:
To open the file, you can do it without skimage, just with what we already have in Odemis. You can write:
``` python
    converter = dataio.find_fittest_converter(fn, default=None, mode=os.O_RDONLY)
    data = converter.read_data(filename)
    image = data[0]
```